glowing-waffles/notebooks/source_detection.py

```diff
@@ -0,0 +1,11 @@
+from ccdproc import CCDData
+from astropy.stats import sigma_clipped_stats
+from photutils import daofind
+
+ccd = CCDData.read('/Users/Nathan/Desktop/Astro-Project/Testing/M52-002R.fit', unit="adu")
+
+def source_detection(ccd):
```

Comment:
Please add a docstring like http://astropy.readthedocs.org/en/stable/development/docrules.html#docstring-standard

We only need the short description, parameters and return sections